lib/trackler/track.rb

```diff
@@ -63,7 +63,7 @@ def icon_path
     end
 
     def icon
-      @icon ||= Image.new(File.join(dir, "img/icon.png"))
+      @icon ||= svg_icon.exists? ? svg_icon : png_icon
```

Comment:
I probably would have called the method `icon_svg` and might have ended up at `icon('svg')` but what you've done is fine too.